spec/features/admin/facility_journals_search_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe "Journals Search" do
+  let(:facility) { create(:setup_facility) }
+  let(:director) { create(:user, :facility_director, facility: facility) }
+  let(:item) { create(:setup_item, facility: facility) }
+  let(:accounts) { create_list(:setup_account, 2) }
+  let(:orders) do
+    accounts.map { |account| create(:complete_order, product: item, account: account) }
+  end
+
+  let(:order_details) { orders.map(&:order_details).flatten }
+
+  before do
+    order_details.each do |detail|
+      detail.update(reviewed_at: 1.day.ago)
+    end
+  end
+
+  it "can do a basic filter" do
```

Comment:
Would it be worth adding a spec around invalid order details or pending journals? Or do we think the existing controller specs will cover that?

Reply:
I think the existing controller specs are fairly thorough there.